src/main/java/com/example/ecommerce/controller/admin/orderController.java

```diff
@@ -0,0 +1,17 @@
+package com.example.ecommerce.controller.admin;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+@Controller
```

Comment:
```suggestion
@Controller("/admin/orders")
```
như ri thì tất cả các action trong controller này đều bắt đầu `/admin/orders`